lib/maid/app.rb

```diff
@@ -155,6 +164,59 @@ def migrate_trash
     end
   end
 
+  def install_launch_agent
+    maid_executable = %x(which maid)
+    plist_destination = File.expand_path("~/Library/LaunchAgents/benjaminoakes.maid.plist")
+    plist_contents = <<-EOF
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN"
+          "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>Label</key>
+    <string>benjaminoakes.maid</string>
+    <key>ProgramArguments</key>
+    <array>
+      <string>/bin/sh</string>
+      <string>-c</string>
+      <string>#{maid_executable} daemon</string>
```

Comment:
Hrm... this could have problems if people are using `rvm` or something else that changes the executable path on occasion... (e.g. Ruby upgrades)

That said, I don't know a better approach off the top of my head.